Gemfile

```diff
@@ -65,7 +65,7 @@ gem 'net-pop' # Remove me after mail upgraded to 2.8.0
 gem 'nokogiri', '~> 1.13'
 gem 'nsa', '~> 0.2'
 gem 'oj', '~> 3.13'
-gem 'ox', '~> 2.14'
+gem 'ox', '~> 2.14', github: 'ohler55/ox', branch: 'develop'
```

Comment:
Isn't this just 2.14.12?
https://github.com/ohler55/ox/commit/cb8f1af3d2191178b413185ce8874244bbb920e5

Reply:
@p8 thanks!  I'll update the Gemfile to point at the released version.